### PR TITLE
Remove all scss and css loaders in build process

### PR DESCRIPTION
**Pull request by YuCJ**
KanbanFlow tasks:
https://kanbanflow.com/t/tZSy8iLk
https://kanbanflow.com/t/MNJcTT18

Change:

- Refactor TopicLandingPage
    - Remove warning and only re-render when slug changed
    - Rewrite des of topic landing page with styled-components
- Remove scss related packages
- Remove unused article-v1 components
- Use normalize.css with static assets rather than package
- Move article container to main chunk

### CODE REVIEW

**Comment by YuCJ** [comment]
Before:

```
[webpack] build client side bundles
NODE_ENV=production node_modules/.bin/webpack --config webpack.config.js --colors
(node:48674) DeprecationWarning: loaderUtils.parseQuery() received a non-string value which can be problematic, see https://github.com/webpack/loader-utils/issues/56
parseQuery() will be replaced with getOptions() in the next major version of loader-utils.
(node:48674) DeprecationWarning: loaderUtils.parseQuery() received a non-string value which can be problematic, see https://github.com/webpack/loader-utils/issues/56
parseQuery() will be replaced with getOptions() in the next major version of loader-utils.
Hash: a7d609d22dfcde86d694
Version: webpack 3.12.0
Time: 38733ms
                                               Asset       Size  Chunks                    Chunk Names
             imagediff-chunk-4c2c961620a82f07bc0a.js     114 kB      13  [emitted]         imagediff
          article-page-chunk-ce4118bbf0ff19b77784.js     317 kB       0  [emitted]  [big]  article-page
            index-page-chunk-c5bab82f5bc044864454.js     253 kB       2  [emitted]  [big]  index-page
            topic-page-chunk-35d2cbd34511ec47a5a6.js     151 kB       3  [emitted]         topic-page
       topic-list-page-chunk-91425374541f8a27ed04.js    84.6 kB       4  [emitted]         topic-list-page
         tag-list-page-chunk-efdb99085354089a0633.js    79.5 kB       5  [emitted]         tag-list-page
    category-list-page-chunk-247f1762bc16e6d88f35.js    79.4 kB       6  [emitted]         category-list-page
           author-page-chunk-1fdc31fa545e63549ba4.js    58.9 kB       7  [emitted]         author-page
      author-list-page-chunk-db7d2b49f2b76081d1e7.js    67.3 kB       8  [emitted]         author-list-page
      photography-page-chunk-850aab388be10de25d3b.js    58.7 kB       9  [emitted]         photography-page
         bookmark-list-chunk-44dd833a8ea2e37e933c.js    29.7 kB      10  [emitted]         bookmark-list
           search-page-chunk-b0ebe356e3dff8a3cd4c.js    1.98 kB      11  [emitted]         search-page
                 main.a7d609d22dfcde86d694.bundle.js     747 kB      12  [emitted]  [big]  main
         about-us-page-chunk-bf7686cba8cbc6c2bd7b.js     326 kB       1  [emitted]  [big]  about-us-page
            blockquote-chunk-9db21f06b3efaa1100ed.js    1.32 kB      14  [emitted]         blockquote
                 audio-chunk-02178ff332dfeea07354.js     154 kB      15  [emitted]         audio
             slideshow-chunk-2f7faee2114728d7c4a9.js    19.4 kB      16  [emitted]         slideshow
               youtube-chunk-e94e0cc1c910dd276804.js    2.08 kB      17  [emitted]         youtube
         unorderedlist-chunk-b048fe7d753b11178987.js  785 bytes      18  [emitted]         unorderedlist
          embeddedcode-chunk-e3a206e4cd1ae4fd3f6f.js    2.55 kB      19  [emitted]         embeddedcode
                lottie-chunk-4f464bd8d47ecac25933.js     151 kB      20  [emitted]         lottie
aboutus-section1-anim3-chunk-3036afbeb7fb043ccc17.js    18.2 kB      21  [emitted]         aboutus-section1-anim3
aboutus-section1-anim2-chunk-a5f1bf9b5607be857a2d.js    18.7 kB      22  [emitted]         aboutus-section1-anim2
aboutus-section1-anim1-chunk-abc28357bd956d3f6fb3.js    11.9 kB      23  [emitted]         aboutus-section1-anim1
             manifest.a7d609d22dfcde86d694.bundle.js     2.4 kB      24  [emitted]         manifest
                       28ca967e059516db8835.main.css    25.3 kB      12  [emitted]         main
```


After:

```
ild client side bundles
NODE_ENV=production node_modules/.bin/webpack --config webpack.config.js --colors
Hash: 575a6a5cc97529d6782e
Version: webpack 3.12.0
Time: 25086ms
                                               Asset     Size  Chunks                    Chunk Names
      author-list-page-chunk-0b4d70f7874615b2430d.js  27.1 kB       8  [emitted]         author-list-page
         about-us-page-chunk-95f1a27ec462037a207b.js   291 kB       0  [emitted]  [big]  about-us-page
            topic-page-chunk-59703762b6c7a73a1e29.js  88.1 kB       2  [emitted]         topic-page
       topic-list-page-chunk-83c1d922bb97b193d4fc.js  60.1 kB       3  [emitted]         topic-list-page
         tag-list-page-chunk-2ca10bc8a3a0d81aa748.js  54.9 kB       4  [emitted]         tag-list-page
    category-list-page-chunk-9d18385cf17d38727618.js  54.8 kB       5  [emitted]         category-list-page
           author-page-chunk-3d301a2d4ad38d921ebb.js  18.8 kB       6  [emitted]         author-page
         bookmark-list-chunk-ae02788c402085d91440.js  28.1 kB       7  [emitted]         bookmark-list
            index-page-chunk-864af67c9c8f35020905.js   219 kB       1  [emitted]         index-page
      photography-page-chunk-358aed332d6637bc02f9.js  26.2 kB       9  [emitted]         photography-page
           search-page-chunk-81db808abc12abfa0cc7.js  1.98 kB      10  [emitted]         search-page
                 main.575a6a5cc97529d6782e.bundle.js  1.06 MB      11  [emitted]  [big]  main
                lottie-chunk-08cb56a4e7db32d7df6b.js   151 kB      12  [emitted]         lottie
aboutus-section1-anim3-chunk-4d8f4ae18c12f9b2a51b.js  18.2 kB      13  [emitted]         aboutus-section1-anim3
aboutus-section1-anim2-chunk-c7d2b7c9c66d762d57e7.js  18.7 kB      14  [emitted]         aboutus-section1-anim2
aboutus-section1-anim1-chunk-9b8a18f5b53e9feeea6a.js  11.9 kB      15  [emitted]         aboutus-section1-anim1
             manifest.575a6a5cc97529d6782e.bundle.js  2.06 kB      16  [emitted]         manifest
```

File: package.json [hunk 1]
```diff
@@ -101,14 +98,11 @@
...
     "pm2": "^2.2.3",
-    "postcss-loader": "^0.9.1",
     "precommit-hook": "^3.0.0",
     "puppeteer": "^1.3.0",
     "redux-mock-store": "^1.0.2",
     "rimraf": "^2.5.2",
-    "sass-loader": "^7.0.3",
     "strip-loader": "^0.1.0",
-    "style-loader": "^0.13.1",
     "url-loader": "^0.5.7",
```
Comment (nickhsine) [inline package.json]: Could we also remove 
`strip-loader`,
`url-loader`,
`extract-text-webpack-plugin`,
`file-loader`,
`json-loader`,
`node-sass`,
`classnames`
?

Comment (nickhsine) [reply package.json]: By the way, there are many packages to remove after we upgrade article component to v2.
If available, we could create another PR to remove them from package.json.

**Comment by nickhsine** [comment]
LGTM

**Comment by YuCJ** [comment]
Updated

**Comment by nickhsine** [comment]
LGTM

**Pull request merged**